game/songs.hh

```diff
@@ -77,11 +77,12 @@ class Songs {
 	size_t loadedSongs() const { return m_songs.size(); }
 
   private:
-  	void LoadCache();
+	void LoadCache();
 	void CacheSonglist();
 
 	class RestoreSel;
-	typedef std::vector<std::shared_ptr<Song> > SongVector;
+	using SongPtr = std::shared_ptr<Song>;
```

Comment:
this kinda using on pointers is usually fishy and here seems really little used.

Reply:
Any suggestion?

Reply:
using unique_ptr?